src/main/java/com/mraof/minestuck/entity/underling/BasiliskEntity.java

```diff
@@ -72,8 +72,8 @@ public static AttributeSupplier.Builder basiliskAttributes()
 	protected void registerGoals()
 	{
 		super.registerGoals();
-		this.goalSelector.addGoal(2, new AnimatedAttackWhenInRangeGoal<>(this, TAIL_SWING_ANIMATION, 0, 3.5F, 40));
-		this.goalSelector.addGoal(2, new AnimatedAttackWhenInRangeGoal<>(this, BITE_ANIMATION, 3.5F, AnimatedAttackWhenInRangeGoal.STANDARD_MELEE_RANGE, 40));
+		this.goalSelector.addGoal(2, new AnimatedAttackWhenInRangeGoal<>(this, TAIL_SWING_ANIMATION, 0, 3.5F, 40, 180.0F, AnimatedAttackWhenInRangeGoal.STANDARD_MELEE_ARC));
+		this.goalSelector.addGoal(2, new AnimatedAttackWhenInRangeGoal<>(this, BITE_ANIMATION, 3.5F, AnimatedAttackWhenInRangeGoal.STANDARD_MELEE_RANGE, 40, 0, 45.0F));
```

Comment:
Ok I actually really like that you can stand on their back now without problems. However there is a (in my opinion) too generous of a region on the front half of their body when standing close to their head where they cannot hurt you at all. See about either slightly expanding the cone or range of either the bite or whip attacks to compensate

Reply:
Sorry to say but I feel the issue is still there. If you walk straight up to the front of the basilisk, you can get closer than where the bite attack activates. You can continue standing inside its head/neck and while the whip animation will play, it does not injure

Reply:
Why would we want the whip animation to injure you there? That doesn't seem like it would be intuitive, since it doesn't match the animation.

Reply:
What I mean is that overall there needs to be adjustments in order to reduce the size of the safe spot. If you dont think increasing the tail whip range/cone makes sense, you can adjust the bite range

Reply:
Oh, I see the problem now. I thought you meant that you were safe when standing behind the head to the right or left, but you can also LITERALLY stand INSIDE the head and be fine, because it won't do the bite attack.

Reply:
Hehe yep! It would probably be a smart choice to add the cone to the canUse() method to also prevent them from repeatedly trying to attack where they cannot

Reply:
Okay, now you will get damaged when standing inside the basilisk's neck, while still keeping the hit-cones mostly accurate to the attack animations. I also made the tail-swipe have a slightly wider arc, so that it isn't _too_ easy to stay in the basilisk's blind-spots.